config.go

```diff
@@ -49,7 +51,7 @@ type DatabaseConfig struct {
 	TLS *TLSConfig
 }
 
-func (c *DatabaseConfig) MySQLConfig() (*mysql.Config, error) {
+func (c DatabaseConfig) MySQLConfig() (*mysql.Config, error) {
```

Comment:
Nothing used the pointer interface. I figure I might as well make this consistent. Shouldn't cause any issues afaict.

Reply:
like justin, i think that we'll need this to be a pointer if we're gonna cache connections in it